### DISCUSSION TITLE
equation number beside equation

### DISCUSSION THREAD

**Discussion by triilman25** [typst/typst#4596] [General]
is there equation number's feature like:
![Screenshot 2024-07-22 072302](https://github.com/user-attachments/assets/af31e0d9-d946-4fed-9daf-80d78b49c724)

**Comment by glyn** [comment]
I have a similar question. I number equations using, for example, `#set math.equation(numbering: "(1)")`. I can even give a name to a particular equation using the `$ ... $ <myname>` syntax. The problem as I see it is that I can't assign names to individual equations which are part of a continuous block and I can't refer to equation names inside a block (as in the example above).

**Reply by laurmaedje** [reply]
Yes, equation numbering and references are unfortunately not yet fully fleshed out. See also https://github.com/typst/typst/issues/380.
